microceph/api/types/services.go

```diff
@@ -20,6 +20,11 @@ type EnableService struct {
 	// Enable Service passes all additional data as a json payload string.
 }
 
+// NFSService holds the Cluster ID of the NFS Service.
```

Comment:
The `EnableService` structure is sufficient for moving payload to the service placement interface. (as used in the cmd package's enable_nfs.go). This structure therefore can be removed (along with the RGW structure below as it is remnant of the past.